scripts/Muon/GUI/Common/home_plot_widget/home_plot_widget_presenter.py

```diff
@@ -107,20 +108,32 @@ def plot_standard_workspaces(self):
 
         self._model.plot(workspace_list, self.get_plot_title())
 
+        workspace_list_inverse_binning = self.get_workspaces_to_plot(self.context.group_pair_context.selected,
+                                                                     not self._view.if_raw(),
+                                                                     self._view.get_selected())
+        self._model.plotted_workspaces_inverse_binning = workspace_list_inverse_binning
+
+        if self.context.fitting_context.fit_list and \
+                any([workspace in workspace_list + workspace_list_inverse_binning
```

Comment:
Can we add a short comment describing what we're trying to achieve in this statement?
I think also having `combined_ws_list = workspace_list + workspace_list_inverse_binning` and then `workspace in combined_ws_list` would be a better solution, just so we don't do `workspace_list + workspace_list_inverse_binning` for every workspace in the loop